salt/journal-cms/config/etc-nginx-sites-enabled-api-dummy-dev.conf.txt

```diff
@@ -0,0 +1,28 @@
+server {
+    listen 8080;
+
+    server_name localhost;
+
+    root /srv/api-dummy/web;
+
+    location = /favicon.ico { access_log off; log_not_found off; }
```

Comment:
we are not going to open port 8080 from outside, so you can skip this `access_log off` directives probably